src/main/java/com/pirimid/cryptotrade/helper/exchange/EXC_Parent.java

```diff
@@ -0,0 +1,21 @@
+package com.pirimid.cryptotrade.helper.exchange;
+
+
+import com.pirimid.cryptotrade.DTO.PlaceOrderReqDTO;
+import com.pirimid.cryptotrade.DTO.PlaceOrderResDTO;
+import com.pirimid.cryptotrade.DTO.SymbolResDTO;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+@Component
+public interface EXC_Parent {
```

Comment:
Can you rename this interface and also will suggest you guys to follow naming convention?